lldb/source/Commands/CommandObjectTarget.cpp

```diff
@@ -2104,9 +2113,10 @@ class CommandObjectTargetModulesDumpSymtab
                 result.GetOutputStream().EOL();
                 result.GetOutputStream().EOL();
               }
-              if (INTERRUPT_REQUESTED(GetDebugger(), 
-                    "Interrupted in dump symtab list with {0} of {1} dumped.", 
-                    num_dumped, num_matches))
+              if (INTERRUPT_REQUESTED(
+                      GetDebugger(),
+                      "Interrupted in dump symtab list with {0} of {1} dumped.",
+                      num_dumped, num_matches))
```

Comment:
Please upstage unrelated code changes. I suspect this is the result of our editor removing trailing whitespace (I do the same) and then clang-format considers this a modified line and formats it.